### ISSUE TITLE
For Cud

### ISSUE THREAD

**Issue by Alexey-T** [Zlitus/sublime-text-timestamp#1]
Hi. Maybe easy to write same (or port) for CudaText?

**Comment by Zlitus** [comment]
Hello! I don’t know so much CudaText but it’s seem very easy to port the plugin. Do you have some experience with that?

**Comment by Alexey-T** [comment]
Yes, it is easy to make same plugin. Doc- howto write plugin - http://synwrite.sourceforge.net/forums/viewtopic.php?f=19&t=739
